python/semantic_kernel/connectors/memory/astradb/astra_client.py

```diff
@@ -7,6 +7,8 @@
 import nest_asyncio
 nest_asyncio.apply()
 
+from semantic_kernel.connectors.memory.astradb.utils import AsyncSession
+
 class AstraClient:
     def __init__(
         self,
```

Comment:
the option to add the session to the init is missing.

Reply:
done